Update phylo tree/run models

ttung opened the pull request:
### Description
1. Add a relationship to express which samples were actually incorporated into a tree.
2. Add data about how the build is made.

Depends on #230 

#### Issue
[ch129581](https://app.clubhouse.io/genepi/story/129581/schema-changes-to-record-which-samples-are-in-a-tree)
[ch130359](https://app.clubhouse.io/genepi/story/130359/schema-changes-to-record-the-build-configuration)

### Test plan
reran the import and pushed the docker image to ECR.

shanaxel42 reviewed (approved):
lgtm

phoenixAja commented on src/py/aspen/database/models/phylo_tree.py:
```diff
@@ -1,13 +1,40 @@
+        primary_key=True,
+    ),
+)
+"""This table records which samples are included in the phylo tree.  Note that this is
+different from which samples are inputs into the build that creates a phylo tree."""
```
can we add this as a docstring to whatever model this is referring to?

ttung replied:
This is applicable to `PhyloTreeSamples`.  It is somewhat a quirk of python docs that the docstring appears below, even in the case of variables.

phoenixAja commented on src/py/aspen/database/models/phylo_tree.py:
```diff
@@ -29,6 +63,19 @@ class PhyloRun(Workflow):
 
+    template_file_path = Column(String, nullable=True)
+    """This is the path, relative to aspen root, for the builds template file.  For all
+    new builds, this should be set.  However, for historically imported """
```
this sentence looks like it was cutoff

ttung replied:
HAHA thanks for noticing this.

ttung replied:
and fixed.

Commit:
Update phylo tree/run models

### Description
1. Add a relationship to express which samples were actually incorporated into a tree.
2. Add data about how the build is made.

#### Issue
[ch129581](https://app.clubhouse.io/genepi/story/129581/schema-changes-to-record-which-samples-are-in-a-tree)
[ch130359](https://app.clubhouse.io/genepi/story/130359/schema-changes-to-record-the-build-configuration)

### Test plan
reran the import and pushed the docker image to ECR.
